adapter/internal/discovery/xds/rate_limiter_cache.go

```diff
@@ -317,7 +331,10 @@ func AddSubscriptionLevelRateLimitPolicy(policyList *types.SubscriptionPolicyLis
 		}
 
 		// Need not to add the Unauthenticated and Unlimited policies to the rate limiter service
-		if policy.Name == "Unauthenticated" || policy.Name == "Unlimited" {
+		if policy.Organization == "carbon.super" && policy.Name == "Unauthenticated" {
+			continue
+		}
+		if policy.Name == "Unlimited" {
```

Comment:
Without using the policy name `Unlimited` to check, shall we use RequestsPerUnit as `-1`?
@malinthaprasan, if a user deletes the default `Unlimited` policy and creates a new one with a different name for the Unlimited policy, do we have to handle this Unlimited case neh?

When creating a policy from admin API, should we have the option to mark a policy as unlimited? or maybe set RequestsPerUnit as -1?